.prettierignore

```diff
@@ -0,0 +1,2 @@
+build
```

Comment:
This file is telling prettier what NOT to lint for styling. See [https://prettier.io/docs/en/ignore.html](https://prettier.io/docs/en/ignore.html)